doc/command_ref.rst

```diff
@@ -141,7 +141,15 @@ Options
     Show the help.
 
 ``--installroot=<path>``
-    set install root
+    Specifies an alternative installroot, relative to where all packages will be
+    installed. Think of this like doing ``chroot <root> dnf`` except using
+    ``--installroot`` allows dnf to work before the chroot is created. The
+    following things are taken from or stored in installroot: cachedir, gpgkey,
```

Comment:
Are you sure that gpgkey is fetched relative to installroot? When I am looking at getKeyForPackage() I strongly suspect that it is not relative to installroot.